v1.0 with GUI and more

Anywhere-Music-Player opened the pull request:
I've used another fork with nice changes, started from there. 
Added GUI and Settings. Icon for app, menu in Status Bar. Control brightness with slider in the menu. And more..

Also, I've added personal option: control Color Temp aka Night Shift, be default its' disabled in the menu.
Using F3 and F4 or slider in menu you can change color temp. I'm personally using it along with brightness. 

![menu](https://user-images.githubusercontent.com/3391456/35537098-bc0d334a-0551-11e8-8527-2b6d7a7ed227.png)![settings](https://user-images.githubusercontent.com/3391456/35537105-bf74b2ba-0551-11e8-81ba-8c8abaaf5e8e.png)


I don't know how to implement multi monitor option, so it's just enabled. But I hope someone can help me with this.. also, I have only 1 monitor, so I can't test this.

jlj committed:
Support for multiple monitors.

- Manage the brightness individually for each connected external monitor.
- Read the brightness from the target monitor if available; otherwise maintain a @[display_id: brightness] dictionary in the user defaults.
- Display the Brightness system OSD on the target monitor.
- Upgraded DDC to a more-recent version.
- Removed useless CoreGraphicsPriv.h.

jlj committed:
Fixed typo in readme

jlj committed:
Better first-launch experience.

If the app is not trusted in the Accessibility preference panel when started, it now waits that the user trusts it before installing the key event monitor. Previously the app had to be relaunched after being authorized to grab the keys, which was not user-friendly at all, because that the app is invisible and had to be killed via the Activity Monitor!!

jlj committed:
Fixed typo in readme

jlj committed:
Fix first launch issue on macOS 10.11 El Capitan and before.

- Waiting for accessibility authorization is now done using the selector-based NSTimer API, making it work on macOS versions < 10.12.
- Updated version information and copyright.

jlj committed:
Show the brightness pane when at min or max level.

- Show the brightness level pane on the target display if the brightness level is not actually changed because already at the min or max level.
- [Internal] Declare C functions and variable as static in AppDelegate.m to better express their expected scope.

jlj committed:
Allow fine brightness adjustment when the option key is pressed.

- Rewritten management of the brightness level, so that possible brightness levels are evenly distributed in the monitor's possible range.
- Added the support of quarter-step adjustments when the Option modifier key is pressed (similar to the system brightness management on the internal display when  Shift+Option is pressed).

Commit:
Clean Xcode files.

Commit:
gitignore

Commit:
Defines BRIGHTNESS_DOWN_KEY and BRIGHTNESS_UP_KEY.

Commit:
WIP: statusBar menu. Cleanup: removed ".xib" file, removed warning, Xcode version check. New: app icon, status bar iocn.

Commit:
Project tweaks.

Commit:
Xcode moved file..

Commit:
WIP: settings and brightness in menu.

Commit:
brightnessSubstepsPerStep is 6, so it's 1% smooth step, more nice.

Commit:
WIP: show % of brightness, change it via slider in menu.

Commit:
UI

Commit:
resize

Commit:
Update Readme.md

Commit:
Update Readme.md

Commit:
tweak size

Commit:
combine

Commit:
Update Readme.md

Commit:
Load/save settings values.

Commit:
Tweaks for width of bar icon and other.

Commit:
Added config file.

Commit:
WIP: Color Temperature feature.

Commit:
v1.0 done. Stable. Testing.

Commit:
Updated preview images.

Commit:
Moved files.

Commit:
Update Readme.md

Commit:
Update preview.

Commit:
Fix detect temp. enabled of startup.

Commit:
Mokup hotkeys and fixes.

Commit:
WIP: added properties

Anywhere-Music-Player commented:
@Bensge Are you interested in this? 
Also, maybe you can add me as contributor? I would love to help, improve GUI and any other things I can.

Commit:
Update Readme.md

Commit:
WIP: keys

Commit:
Improve done.

Commit:
Merge branch 'improve'

* improve:
  Improve done.
  WIP: keys
  WIP: added properties
  Mokup hotkeys and fixes.

Commit:
Updated readme & settings. Minimum project target is 10.12.4

Commit:
No warnings.

Anywhere-Music-Player commented:
@Bensge Hello :) ?

gregcotten commented:
This looks great! Seems like with at least Mojave you can adjust the color temperature of the monitor in the Display settings.

Commit:
Update Readme.md

jactor-sue commented:
Great work !

Commit:
Small project upgrades

Commit:
Update Readme.md

Commit:
Build successful using Xcode 12.

Commit:
Update Readme.md

Commit:
Just re-compile:
 - Xcode 13.3
 - macOS 12.3.1
 - Apple Silicon(please note: app will not work for MacBook's screens)
 - Apple notary service!
